### ISSUE TITLE
consider renaming diag argument in psychsyn

### ISSUE THREAD

**Issue by awmeade** [ryentes/careless#30]
Great package!  So nice it's on CRAN now!  
I was expecting the diag argument to mean 'diagonal' but i can't tell that there are any diagonals involved.  perhaps renaming to something more intuitive?

**Comment by ryentes** [comment]
Yeah in retrospect that's an ambiguous abbreviation. We were discussing several alternative names for that argument. for example: "n_pairs", "count_pairs", or simply "pairs". The output column name is currently numPairs but that may change too as we have moved away from camel case in our more recent code.

I definitely welcome comments and suggestions.
